src/snap_storage.test.ts

```diff
@@ -120,4 +120,33 @@ describe("Snapshot storage", () => {
 
     assert(!storedSnap.isFresh, "Existing snapshots should not be fresh");
   });
+
+  it("caches a modified fetched JSON response", async () => {
+    const testFile = "test_data/mutator.json";
```

Comment:
I wasn't sure what else to do here as `deno.json` is already cached by the previous test.